### DISCUSSION TITLE
Having trouble uploading files

### DISCUSSION THREAD

**Discussion by salmgr87** [community/community#39851] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I appreciate anyone's help or suggestions. My M1 MacBook Air suddenly is not letting me:

1. Create new repositories in the browser (the "create" button is greyed out).
2. Upload files directly to a repository in the browser (either as a new file or a new commit). The browser doesn't respond to dragging files, nor to the "choose your files" window after I choose the files.

I've tried on Chrome (updated) and Safari (updated). I'm using my work computer to complete the Odin Project, so I'm unable to use Terminal to commit my changes. The browser method was my workaround, but it's suddenly not working. Any suggestions would be most appreciated. The site works perfectly fine on my mobile device, so it's not my account.

Thanks!

**Comment by ossanna16** [comment]
Hi there @salmgr87 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
